docs/ref/settings.txt

```diff
@@ -2764,11 +2764,11 @@ See also :setting:`LANGUAGE_CODE`, :setting:`USE_I18N` and :setting:`USE_TZ`.
 Default: ``False``
 
 A boolean that specifies whether to display numbers using a thousand separator.
-When :setting:`USE_L10N` is set to ``True`` and if this is also set to
-``True``, Django will use the values of :setting:`THOUSAND_SEPARATOR` and
-:setting:`NUMBER_GROUPING` to format numbers unless the locale already has an
-existing thousands separator. If there is a thousands separator in the locale
-format, it will have higher precedence and will be applied instead.
+When set to ``True`` and :setting:`USE_L10N` is also ``True``, Django will use
+the values of the locale's ``THOUSAND_SEPARATOR`` format (if defined), the
+:setting:`THOUSAND_SEPARATOR` setting and the :setting:`NUMBER_GROUPING`
```

Comment:
```suggestion
the values of the locale's ``THOUSAND_SEPARATOR`` format (if defined), or the
:setting:`THOUSAND_SEPARATOR` setting and the :setting:`NUMBER_GROUPING`
```

Reply:
Yes but this needs movement of the comma...

Reply:
Indeed. It would need to be before the "and". I thought about that, but I'm not sure it makes it clear enough to drop the subsequent sentence.

Reply:
👍